staging/src/k8s.io/client-go/discovery/round_tripper.go

```diff
@@ -47,4 +47,13 @@ func (rt *cacheRoundTripper) RoundTrip(req *http.Request) (*http.Response, error
 	return rt.rt.RoundTrip(req)
 }
 
+func (rt *cacheRoundTripper) CancelRequest(req *http.Request) {
+	type canceler interface {
+		CancelRequest(*http.Request)
+	}
+	if cr, ok := rt.rt.Transport.(canceler); ok {
+		cr.CancelRequest(req)
+	}
```

Comment:
you need to give the same kind of message as the others when this fails.  Link to an example when you add it.  Otherwise we'll have another incomplete step in the chain.

Reply:
Added a log, following example from https://github.com/kubernetes/client-go/blob/master/transport/round_trippers.go#L169